apps/core/templates/core/_maintenance_or_offline.html

```
@@ -1,32 +1,32 @@
+    <h1 class="text-center">🚧 🚧</h1>
+    <hr>
+    <p class="text-center">Please check back later!</p>
+    {% if is_in_maintenance %}
+      <p class="text-center text-muted">We are sorry for any inconveniences</p>
+    {% elif called_by_offline_view %}
+      <p class="text-center text-muted">yamsa needs internet to work</p>
     {% endif %}
+  </div>
+  {% if called_by_offline_view %}
+    <div class="fixed-bottom z-0 d-flex justify-content-center mb-5">
+      <button class="btn btn-primary"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)